File: pdlearning/src/web-apps/src/libraries/domain-api/src/learner/models/user-bookmarked.model.ts
export interface IUserBookMarkedModel {
  itemId: string;
  countTotal: number;
}

export class UserBookMarkedModel implements IUserBookMarkedModel {
  public itemId: string;
  public countTotal: number = 0;

  constructor(data?: IUserBookMarkedModel) {
    if (data == null) {
      return;
    }
    this.itemId = data.itemId;
    this.countTotal = data.countTotal;
  }
}

File: pdlearning/src/microservices/learner-microservice/Api/Application/Models/UserBookmarkedModel.cs
using System;

namespace Microservice.Learner.Application.Models
{
    public class UserBookmarkedModel
    {
        public UserBookmarkedModel()
        {
        }

        public UserBookmarkedModel(Guid itemId, int totalCount)
        {
            CountTotal = totalCount;
            ItemId = itemId;
        }

        public Guid ItemId { get; set; }

        public int CountTotal { get; set; }
    }
}
